data_structures/src/chain.rs

```diff
@@ -1873,7 +1888,7 @@ impl TransactionsPool {
     /// // TODO: assert!(tx1.weight() >= tx2.weight());
```

Comment:
Remove this TODO, we would need to assert that `tx1.fee() / tx1.weight() >= tx2.fee() / tx2.weight()` but that's not easy.
```suggestion
```